### PR TITLE
update kornia_rs and make sure contiguous data

### PR DESCRIPTION
**Pull request by edgarriba**
#### Changes

- update kornia-rs to 0.1.0 https://docs.rs/crate/kornia-rs/0.1.0
  - now images come as numpy array already
- make kornia_rs mandatory
- add `force_contiguous` to `tensor_to_image` to avoid panic in konia_rs crate

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

**Review by johnnv1 — APPROVED** [review]
Does `kornia-rs `have a bin for all platforms/os?

**Comment by edgarriba** [comment]
> Does `kornia-rs `have a bin for all platforms/os?

Yes for at least the ones we support for kornia pytorch

**Review by gau-nernst — APPROVED** [review]

File: kornia/io/io.py [hunk 1]
```diff
@@ -30,28 +25,42 @@ class ImageLoadType(Enum):
...
-    # the obtained stream bits are encapusalted to a cv::Tensor data structure without
-    # memory ownership and passed as PyCapsule from rust to python.
-    cv_tensor = kornia_rs.read_image_rs(path_file)
-    # for convenience use the torch dlpack parser to get a zero copy torch.Tensor
-    # TODO: evaluate other potential API so that we can return in numpy, jax, mxnet since
-    # the kornia_rs cv::Tensor has this ability.
-    th_tensor = dlpack.from_dlpack(cv_tensor)  # HxWx3
-    # move the tensor to the desired device, move the data layout to CHW and clone
-    # to return an owned data tensor.
+def load_image_to_tensor(path_file: Path, device: Device) -> Tensor:
```
Comment (johnnv1) [inline kornia/io/io.py:28]: needs docs entry

Comment (edgarriba) [reply kornia/io/io.py:28]: this is intended to be private function

**Pull request merged**